src/main.rs

```diff
@@ -371,32 +381,54 @@ fn filter_commits<'repo>(config: &Config, commits: Vec<Commit<'repo>>) -> Vec<Co
 struct CommitHours {
     email: Option<String>,
     author_name: Option<String>,
+    breakdown: HashMap<String, Duration>,
     duration: Duration,
     commit_count: usize
 }
 
-fn estimate_author_time(mut commits: Vec<Commit>, email: Option<String>, max_commit_diff: &Duration, first_commit_addition: &Duration) -> CommitHours {
+fn estimate_author_time(mut commits: Vec<Commit>, email: Option<String>, max_commit_diff: &Duration, first_commit_addition: &Duration, display_breakdown: &bool) -> CommitHours {
     let author_name = commits[0].author().name().map(|n| n.to_string());
 
     commits.sort_by(|a, b| a.time().cmp(&b.time()));
 
+    //let lalala = Utc.timestamp(commits[commits.len() - 3].time().seconds(), 0).format("%Y-%m-%d");
```

Comment:
Could we remove this if we're not using it?

Reply:
Terribly sorry, accidentally committed debug prints!